### PR TITLE
sf RNs: May 17

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/sf/README.md [hunk 1]
```diff
@@ -21,10 +21,46 @@ Additional documentation:
...
+
+* FIX: The `orgID` value in the JSON output resulting from running `sf org create scratch <flags> --json` now contains the actual scratch org ID (starts with 00D) rather than the ScratchOrgInfo record ID (starts with 2SR). (GitHub issue [#2131](https://github.com/forcedotcom/cli/issues/2131), plugin-org PR [#675](https://github.com/salesforcecli/plugin-org/pull/675))
+
+* FIX: All commands now know about the `org-metadata-rest-deploy` configuration variable, which is the new `sf`-style name for the `restDeploy` configuration variable. (GitHub issue [#2127](https://github.com/forcedotcom/cli/issues/2127), sfdx-core PR [#834](https://github.com/forcedotcom/sfdx-core/pull/834), plugin-signups PR [#276](https://github.com/salesforcecli/plugin-signups/pull/276))
+
+* FIX: The `project deploy|retrieve start` commands now support the UserAccessPolicy metadata type.
```
Comment (mshanemc) [inline releasenotes/sf/README.md]: *everything* supports that type now.  Could we say it in a more generic way about the CLI, generally?

File: releasenotes/sf/README.md [hunk 2]
```diff
@@ -21,10 +21,46 @@ Additional documentation:
...
+
+* FIX: The `orgID` value in the JSON output resulting from running `sf org create scratch <flags> --json` now contains the actual scratch org ID (starts with 00D) rather than the ScratchOrgInfo record ID (starts with 2SR). (GitHub issue [#2131](https://github.com/forcedotcom/cli/issues/2131), plugin-org PR [#675](https://github.com/salesforcecli/plugin-org/pull/675))
+
+* FIX: All commands now know about the `org-metadata-rest-deploy` configuration variable, which is the new `sf`-style name for the `restDeploy` configuration variable. (GitHub issue [#2127](https://github.com/forcedotcom/cli/issues/2127), sfdx-core PR [#834](https://github.com/forcedotcom/sfdx-core/pull/834), plugin-signups PR [#276](https://github.com/salesforcecli/plugin-signups/pull/276))
+
+* FIX: The `project deploy|retrieve start` commands now support the UserAccessPolicy metadata type. 
+
+* FIX: You can now run `project deploy start --metadata-dir`, which deploys source in metadata format, from outside a Salesforce DX project. (GitHub issue [#2089](https://github.com/forcedotcom/cli/issues/2089), plugin-deploy-retrieve PR [#619](https://github.com/salesforcecli/plugin-deploy-retrieve/pull/619))
```
Comment (mshanemc) [inline releasenotes/sf/README.md]:
```suggestion
* FIX: You can now run `project deploy start --metadata-dir`, which deploys source in metadata format, from outside a Salesforce DX project. (GitHub issue [#2089](https://github.com/forcedotcom/cli/issues/2089), plugin-deploy-retrieve PR [#619](https://github.com/salesforcecli/plugin-deploy-retrieve/pull/619)).  While we were at it, we made `project retrieve start` work outside a project, too, when using `target-metadata-dir`. 
```

Comment (mshanemc) [reply releasenotes/sf/README.md]: or you can decide if that should be its own 'NEW'

File: releasenotes/sf/README.md (hunk 1 above)
Comment (jshackell-sfdc) [reply releasenotes/sf/README.md]: Sure.  How about just "Salesforce CLI now supports the blahdiblah md type"?

Comment (mshanemc) [inline releasenotes/sf/README.md]:
```suggestion
* NEW: Projects now support the UserAccessPolicy metadata type. 
```

**Review by mshanemc — APPROVED** [review]

File: releasenotes/sf/README.md [hunk 3]
```diff
@@ -21,10 +21,46 @@ Additional documentation:
 * [Salesforce CLI Plugin Developer Guide (sf)](https://github.com/salesforcecli/cli/wiki/Quick-Introduction-to-Developing-sf-Plugins)
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
-## 1.78.0 (May 17, 2023) [stable-rc]
+## 1.79.0 (May 24, 2023) [stable-rc]
+
+ANNOUNCEMENT: If you install Salesforce CLI using `npm`, and use Node.js 14 or 16, be aware of these [end-of-life dates](https://github.com/forcedotcom/cli/issues/1985).
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
```
Comment (mshanemc) [inline releasenotes/sf/README.md:29]:
```suggestion

* NEW: the `project deploy` commands now leave the test-level undefined by default.  This lets the org decide what tests need to run.  If you don't specify the flag,
  - non-production orgs won't run any tests
  - production orgs will run tests if the deployment includes some apex 
```

**Pull request merged**